blimp-live-widgets/src/components/GoogleCalendar/index.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Calendar, momentLocalizer } from 'react-big-calendar'
+import moment from 'moment'
+import '!style-loader!css-loader!react-big-calendar/lib/css/react-big-calendar.css'
```

Comment:
What does this line do? Is this the correct way to import? Some suggestions here: https://github.com/intljusticemission/react-big-calendar/issues/234 but idk if any of them are the best choice, just wondering if this is the best way to go about it.